src/hapi_schema/db_refugees.py

```diff
@@ -0,0 +1,97 @@
+"""Refugees and persons of concern table and view."""
+
+# Note: this one is a bit tricky, because the view has to join with
+# the location table twice
+
+from datetime import datetime
+
+from sqlalchemy import (
+    DateTime,
+    Enum,
+    ForeignKey,
+    Integer,
+    String,
+    select,
+)
+from sqlalchemy.orm import Mapped, aliased, mapped_column, relationship
+
+from hapi_schema.db_location import DBLocation
+from hapi_schema.utils.base import Base
+from hapi_schema.utils.constraints import (
+    population_constraint,
+    reference_period_constraint,
+)
+from hapi_schema.utils.enums import Gender, PopulationGroup
+from hapi_schema.utils.view_params import ViewParams
+
+
+class DBRefugees(Base):
+    __tablename__ = "refugees"
+    __table_args__ = (
+        reference_period_constraint(),
+        population_constraint(population_var_name="population"),
```

Comment:
You can add min and max age constraints here (they are in the constraints utils)